examples/dummy_plugin/dummy_plugin/forms.py

```diff
@@ -0,0 +1,43 @@
+from django import forms
+
+from nautobot.utilities.forms import (
+    BootstrapMixin,
+    BulkEditForm,
+    CSVModelForm,
+)
+
+from .models import DummyModel
+
+
+class DummyModelForm(BootstrapMixin, forms.ModelForm):
+    """Generic create/update form for `DummyModel` objects."""
+
+    class Meta:
+        model = DummyModel
+        fields = ["name", "number"]
+
+
+class DummyModelCSVForm(CSVModelForm):
+    """Generic CSV bulk import form for `DummyModel` objects."""
+
+    class Meta:
+        model = DummyModel
+        fields = DummyModel.csv_headers
+
+
+class DummyModelFilterForm(BootstrapMixin, forms.Form):
+    """Filtering/search form for `DummyModel` objects."""
+
+    model = DummyModel
+    q = forms.CharField(required=False, label="Search")
+    name = forms.CharField(max_length=20, required=False)
+    number = forms.IntegerField(required=False)
+
+
+class DummyModelBulkEditForm(BootstrapMixin, BulkEditForm):
+    """Bulk edit/delete form for `DummyModel` objects."""
```

Comment:
I don't think the `BulkDeleteView` actually uses this form?

Reply:
Ah yeah that was just copypasta.